digiwf-email-integration/src/main/java/io/muenchendigital/digiwf/email/integration/domain/service/MailingService.java

```diff
@@ -0,0 +1,88 @@
+package io.muenchendigital.digiwf.email.integration.domain.service;
+
+import io.muenchendigital.digiwf.email.integration.domain.exception.MissingInformationMailException;
+import io.muenchendigital.digiwf.email.integration.domain.model.Mail;
+import io.muenchendigital.digiwf.s3.integration.client.repository.DocumentStorageFileRepository;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.tika.Tika;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.mail.javamail.MimeMessagePreparator;
+
+import javax.mail.Message;
+import javax.mail.internet.InternetAddress;
+import javax.mail.util.ByteArrayDataSource;
+
+@Slf4j
+@AllArgsConstructor
+public class MailingService {
+
+    private final JavaMailSender mailSender;
+    private final String fromAdress;
+    private final DocumentStorageFileRepository documentStorageFileRepository;
+
+    /**
+     * Send a mail.
+     *
+     * @param mail mail that is sent
+     */
+    public void sendMail(final Mail mail) throws MissingInformationMailException {
+        final StringBuilder exceptionText = new StringBuilder();
+        if (!StringUtils.isNotEmpty(mail.getReceivers())) {
+            exceptionText.append("No receivers given. ");
+        }
+        if (!StringUtils.isNotEmpty(mail.getSubject())) {
+            exceptionText.append("No subject given. ");
+        }
+        if (!StringUtils.isNotEmpty(mail.getBody())) {
+            exceptionText.append("No body given. ");
+        }
+
+        if (StringUtils.isNotEmpty(exceptionText)) {
+            throw new MissingInformationMailException(exceptionText.toString());
+        }
+
+            //handler
+            final MimeMessagePreparator preparator = mimeMessage -> {
```

Comment:
wrong indentation

Reply:
Just use the IntelliJ Save Actions Plugin and activate it.

Reply:
Done and added "Reformat file" to Save Actions.